pkg/rbac/parser_integration_test.go

```diff
@@ -52,7 +52,7 @@ var _ = Describe("ClusterRole generated by the RBAC Generator", func() {
 			Expect(err).NotTo(HaveOccurred())
 
 			By("parsing the desired YAML")
-			for i, expectedRoleBytes := range bytes.Split(expectedFile, []byte("\n---\n"))[1:] {
+			for i, expectedRoleBytes := range bytes.Split(expectedFile, []byte("\n---\n")) {
```

Comment:
Note: this change allows to use the directly generated output as golden file. The `[1:]` is not necessary anymore because the document starts with `---\n`